.github/workflows/dapp_tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Run Dapp Tests
+
+# Only runs when manually triggered
+on:
+  workflow_dispatch:
+
+jobs:
+  run-tests-devnet:
+    name: Run Tests on Devnet
+    runs-on: ubuntu-latest
+    environment:
+      name: devnet
+    needs: run-tests-testnet
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run Dapp Tests Script on Devnet
+        run: |
+          chmod +x ./integration_test/dapp_tests/dapp_tests.sh
```

Comment:
can we just do `chmod +x` to the file in git so we don't have to run this every time?

Reply:
Will iterate on this in the next PR after merging this to main so I can test the workflow